### DISCUSSION TITLE
Delay in ReadingLogBufferAndSendingHttp configurable

### DISCUSSION THREAD

**Discussion by thgau** [tkowalcz/tjahzi#76] [Q&A] [answered]
The thread ReadingLogBufferAndSendingHttp does a busy waiting loop with a delay of 1 ms. Is the delay somehow configurable? I would like to increase it...

**Comment by tkowalcz** [comment, marked as answer]
Yes, I see how that might be a problem. I'll make it configurable.

**Reply by tkowalcz** [reply]
I have implemented it in https://github.com/tkowalcz/tjahzi/issues/77 and released as part of `0.9.18`.
